source/content/guides/decoupled/overview/07-support.md

```diff
@@ -27,7 +27,7 @@ Technical Support covers issues related to the following:
 - Troubleshoot customer cases related to the platform
 - Enable customers by providing related documentation, where to access logs, and when appropriate share similar case studies that provide a solution for the customer in line with the product life cycle.
 - Advocate for customers by validating and reporting bugs, feature requests, and improving platform documentation in line with the product life cycle
-- During customer outages that cannot be resolved by the customers or CSE, escalate to on-call engineers
+- For business-critical incidents or outages that cannot be resolved by the customers or their Customer Success Engineer (CSE), escalate the issue to our engineering team by opening a general support or an emergency ticket (for **Diamond** and **Platinum** customers).
```

Comment:
Two pieces of feedback here:
- we shouldn't use "CSE" in our public facing docs, it's an internal term. 
- it feels like we have a lot of `or` statements (both clauses and exceptions), which can make it hard to keep track of the logic

### CSE terminology
The usage of "Customer Success Engineer" and "engineering team" could cause confusion for our customers, because many are not familiar with Pantheon's internal job titles, (and the word engineer is used in both). We also never use "CSE" anywhere else in the docs. Instead, we can just refer to CSEs as "support", avoiding the acronym. 

**Suggestion**
change `their Customer Success Engineer (CSE)` to `Pantheon support`.

### Many "or"s
We have three "ors" in one sentence here. The final one is an exception clause, informing the reader that emergency tickets are only for Platinum/Diamond customers, but it could be interpreted as "general support or emergency tickets are both limited to certain customers.". 

**Suggestion:**
Move the emergency clause to its own sub-bullet point:
- For business-critical incidents or outages that cannot be resolved by the customers or Pantheon support, escalate the issue to our engineering team by opening a general support ticket or an emergency ticket.
  - *Emergency tickets are only available to Platinum and Diamond customers.*